static/sass/components/_author.scss

```diff
@@ -0,0 +1,108 @@
+// Author Box
+// ==========
+
+[data-module] {
+  + [data-module='rst-author'] {
```

Comment:
I wrote it like this to be more specific than the 
```
  [data-module] {
    + [data-module] {}
  }
```
but maybe using the attribute with the name is more specific without the parent? I feel like that hasn't worked in the past, but I could be wrong. There is something that I come across where specificity that I expect when using data attributes doesn't work out for me. I have to nail down what that is next time I come across it.